roles/openshift_cloud_provider/templates/cloud.conf.j2

```diff
@@ -0,0 +1,8 @@
+[Global]
```

Comment:
This file should probably be roles/openshift_cloud_provider/templates/openstack/cloud.conf.j2 or roles/openshift_cloud_provider/templates/openstack-cloud.conf.j2 instead to avoid naming conflicts as other providers are added.